docs/guides/serverless-port-code.mdx

```diff
@@ -173,14 +169,13 @@ pass_manager = generate_preset_pass_manager(
 @distribute_task(target={"cpu": 1})
 def transpile_parallel(circuit, pass_manager):
     """Distributed transpilation for an abstract circuit into an ISA circuit for a given backend."""
-    
+
     isa_circuit = pass_manager.run(circuit)
 
     return isa_circuit
 
 try:
     # Get backend
-    # Follow instructions at https://quantum.cloud.ibm.com/docs/guides/cloud-setup to authenticate with your API key.
     service = QiskitRuntimeService(channel="ibm_cloud")
```

Comment:
Ditto if channel should be removed. It'd be good to ripgrep for `channel= docs/guides` - maybe create a new issue?

Reply:
No need to remove.